plugins/modules/create_blueprint.py

```diff
@@ -80,6 +81,15 @@
         type: dict
         default: {}
         required: false
+    containers:
+        description:
+            - List of container images to include in compose
+            - Source must be defined, name and tls-verify are optional
+            - More information can be found here: https://www.osbuild.org/guides/image-builder-on-premises/blueprint-reference.html#containers
```

Comment:
Need to quote this or the doc rendering will fail because of fun docstring -> yaml string conversion issues.

Basically just make it:
```python
            - 'More information can be found here: https://www.osbuild.org/guides/image-builder-on-premises/blueprint-reference.html#containers'

```

And the sanity tests will stop failing

Reply:
Fixed in https://github.com/redhat-cop/infra.osbuild/pull/285/commits/ed8196a7e5705c4a9576d754a7c7d5abafa72ac7